src/test/java/it/pagopa/interop/probing/eservice/operations/integration/repository/EserviceViewRepositoryTest.java

```diff
@@ -68,7 +68,7 @@ void testFindAll_whenExistsEservicesOnDatabase_thenReturnTheListNotEmpty() {
 	@DisplayName("the retrieved list of e-services is empty")
 	void testFindAll_whenNotExistsEservicesOnDatabase_thenReturnTheListEmpty() {
 		List<EserviceState> listEservice = new ArrayList<>();
-		listEservice.add(EserviceState.ACTIVE);
+		listEservice.add(EserviceState.ONLINE);
```

Comment:
Use immutable such List.of(EserviceState.ONLINE)

Reply:
Done, I fixed it